truffle/test/Splitter.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const Splitter = artifacts.require('Splitter');
+const truffleAssert = require('truffle-assertions');
+const BN = web3.utils.BN;
+
+contract('Splitter', (accounts) => {
+    let instance;
+
+    // Initiating new contract before each test
+    beforeEach("Creating new contract", async () => {
+        instance = await Splitter.new({from: accounts[0]});
+    });
+
+    //Testing the balance update
+    it('Balance should be equal to 1 ether', () => {
+        return instance.splitEth(accounts[1], accounts[2], {from: accounts[0], value: web3.utils.toWei("1", "ether")})
```

Comment:
`"0.1", "ether"`

Reply:
I don't get your point here. The purpose of this test is to send 1 ether not 0.1

Reply:
Yes. But I think it is still too high a value for a test.